web/src/app/alerts/components/AlertDetails.js

```diff
@@ -122,26 +122,40 @@ export default class AlertDetails extends Component {
     })
   }
 
+  renderUsers(users, stepID) {
+    return users.map((user, i) => {
+      const sep = i === 0 ? '' : ', '
+      return (
+        <span key={stepID + user.id}>
+          {sep}
+          {UserLink(user)}
+        </span>
+      )
+    })
+  }
+
   /*
    * Returns properties from the escalation policy
    * for easier use in functions.
    */
   epsHelper() {
-    const eps = this.props.data.escalation_policy_snapshot
+    const ep = this.props.data.service.escalationPolicy
     const alert = this.props.data
+    const state = this.props.data.state
+
     return {
-      repeat: eps.repeat,
-      numSteps: eps.steps.length,
-      steps: eps.steps,
+      repeat: state?.repeatCount,
+      numSteps: ep.steps.length,
+      steps: ep.steps,
       status: alert.status.toLowerCase(),
```

Comment:
Now that we're using the new API, maybe we should get rid of the use of `.toLowerCase()` everywhere for status comparisons.

That way every usage has identical values (e.g. `StatusUnacknowledged`).